src/main/resources/yamlschema/BatchResponse.yaml

```diff
@@ -0,0 +1,25 @@
+#version: 0.0.3
+title: BatchResponse
+type: object
+description: Represents the response from the print provider in response to recieving a batch file from EROP
+properties:
+  batchId:
+    type: string
+    example: 1000-2000
```

Comment:
Is this a good example?

Reply:
No, it's a terrible example. But it's what was in the original spec file (`PrintProviderSchema.yaml`) that Krister has put together with the Print Provider.
The scope of my work here was getting it to a point of being able to codegen it. I think there will be some subsequent refinement of these specs over the next week or so to get them into a state that we are happy with, but that is not this ticket.